plugins/LayerView/LayerViewProxy.py

```diff
@@ -36,6 +37,12 @@ def currentLayer(self):
         if type(active_view) == LayerView.LayerView.LayerView:
             return active_view.getCurrentLayer()
 
+    @pyqtProperty(int, notify = currentLayerChanged)
+    def minimumLayer(self):
+        active_view = self._controller.getActiveView()
+        if type(active_view) == LayerView.LayerView.LayerView:
```

Comment:
I copied the method how current properties work. Maybe if we add another layer view we can change it. For now it doesn't matter.